src/color-picker/color-picker.js

```diff
@@ -130,8 +129,11 @@ const Self = class ColorPicker extends NudeElement {
 
 		color: {
 			type: Color,
+			get () {
+				return this.defaultColor;
+			},
 			set (value) {
-				this.defaultColor = new Color(value).to(this.space);
+				this.defaultColor = value;
```

Comment:
We don't need `new Color(value).to(this.space)` anymore because `defaultColor.convert()` has our back.

Reply:
Wait, is `color` and `defaultColor` really the same thing? If so, why do we even have two props?

Reply:
We don't need two, indeed. IDK why I kept them (even though I was thinking of merging them). 🤷‍♂️
Now, I merged them.